src/attributes.h

```diff
@@ -0,0 +1,5 @@
+#include "pkcs11-compat.h"
```

Comment:
It's harmless here, but usually headers should have an inclusion guard, no? I have seen `#pragma once` used somewhere in the project.